apps/frontend/src/components/fileBrowser/FileTable.vue

```diff
@@ -1,6 +1,7 @@
 <template>
     <v-contextmenu ref="contextmenu-filebrowser-item-local">
         <v-contextmenu-item
+            v-if="!currentRightClickedItem.data?.isDirectory"
```

Comment:
might add another check? is it possible to just drag a folder? if yes, please add this check to the @drag event